components/widgets/icons/login_gitlab_icon.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+export default function LockCircleSolidIcon(props: React.HTMLAttributes<HTMLSpanElement>) {
+    const {formatMessage} = useIntl();
```

Comment:
nit: wondering if for all these icons instead of formating the aria-label message it sould make more sense to pass that value as a prop and avoid the extra import and also they would be more "customizable"